frontend/src/components/VMediaInfo/VMediaTags.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <ul v-if="filteredTags.length" class="flex flex-wrap gap-2">
+    <VMediaTag v-for="(tag, index) in filteredTags" :key="index" tag="li">{{
+      tag.name
+    }}</VMediaTag>
+  </ul>
```

Comment:
I don't mind but I want to know why this is a separate component. Is there potential to reuse it in other places?

Reply:
I think I extracted it before combining the `VAudioDetails` and `VImageDetails` when trying to add the external icon links to the source links in the metadata section. Then, this PR grew and changed, so this component might not be necessary anymore.
It does remove the `filteredTag` function from the parent component, making it a bit smaller. Although, in the future, the tags should be cleaned up in the catalog, and this function will not be necessary anymore.

Reply:
I moved the tag filtering to `decodeMediaData`, where all media items from the API are cleaned for the frontend.